File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.js
// Last updated: 10/03/2026, 10:02:56
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let X=0;
    for(let i=0;i<operations.length;i++){
        if(operations[i].includes("+")){
            X++
        }
        else{
            X--;
        }
    }
    return X;
};

File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.cpp
// Last updated: 03/08/2026, 10:07:58
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {

        int X = 0;

        for (int i = 0; i < operations.size(); i++) {

            if (operations[i].find('+') != string::npos) {
                X++;
            } 
            else {
                X--;
            }
        }

        return X;
    }
};